src/sdg/producten/models/product.py

```diff
@@ -79,6 +84,52 @@ def upn_uri(self):
     def upn_label(self):
         return self.upn.upn_label
 
+    @property
+    def upn_is_verweiderd(self):
```

Comment:
```suggestion
    def upn_is_verwijderd(self):
```

Maar wellicht deze functie schrappen? Of je nu doet `p.upn_is_verwijderd` of `p.upn.is_verwijderd` maakt niet zo veel uit. Less code is better :)